src/commands/utility/changeStatus.js

```diff
@@ -34,7 +35,9 @@ module.exports = {
       const newStatus = MAPPED_STATUS_COMMANDS[status];
 
       if (!newStatus) {
-        return await interaction.reply('Invalid status command.');
+        return await interaction.reply(
+          translateLanguage('changeStatus.invalidStatus')
+        );
```

Comment:
same comment here about errors ,group it in a property called "error", if they are specific of the feature then save it under feature property, if not, if it is general, just move it to a new property called "error". In this case I think it should be `"changeStatus.errors.invalidStatus"`, what do you think?